src/Bridge/Laravel/Providers/HttpClientServiceProvider.php

```diff
@@ -28,8 +29,8 @@ public function register(): void
         $this->app->singleton(ExceptionHandlerInterface::class, ExceptionHandler::class);
 
         // Concrete implementations
-        $this->app->singleton(Client::class);
-        $this->app->singleton(LoggingClient::class, static function (Container $app) {
+        $this->app->singleton(ClientInterface::class, LoggingClient::class);
+        $this->app->singleton(LoggingClient::class, static function (Container $app): LoggingClient {
             return new LoggingClient(
```

Comment:
This allows  the `ClientInterface::class` abstract to be overridable in tests, sets the `LoggingClient` as default for anything asking for a `ClientInterface`, and doesn't break anything that's asking for a `LoggingClient` as a concrete class.